packages/target-ethers-v5-test/types/Events.d.ts

```diff
@@ -93,6 +93,32 @@ interface EventsInterface extends ethers.utils.Interface {
   getEvent(nameOrSignatureOrTopic: "NoArgsEvent"): EventFragment;
 }
 
+export type AnonEvent1_TypedEvent = TypedEvent<
+  [BigNumber] & { value1: BigNumber }
+>;
+
+export type Event1_TypedEvent = TypedEvent<
```

Comment:
Can we simply use `AnonEvent1Event` instead of `AnonEvent1_TypedEvent` in a happy case? Where there are overloads let's use `Event3_bool_uint256_Event` instead of `Event3_bool_uint256_TypedEvent`. So basically just drop `Typed`.